src/EnergyPlus/DemandManager.cc

```diff
@@ -1079,6 +1092,116 @@ namespace DemandManager {
+
+				DemandMgr(MgrNum).Type = ManagerTypeVentilation;
+
+				if (!lAlphaFieldBlanks(2)) {
+					DemandMgr(MgrNum).AvailSchedule = GetScheduleIndex(AlphArray(2));
+
+					if (DemandMgr(MgrNum).AvailSchedule == 0) {
+						ShowSevereError(CurrentModuleObject + "=\"" + cAlphaArgs(1) + "\" invalid " + cAlphaFieldNames(2) + "=\"" + AlphArray(2) + "\" not found.");
+						ErrorsFound = true;
+					}
+				}
+				else {
+					DemandMgr(MgrNum).AvailSchedule = ScheduleAlwaysOn;
+				}
+
+				// Validate Limiting Control
+				{ auto const SELECT_CASE_var(AlphArray(3));
+				if (SELECT_CASE_var == "OFF") {
+					DemandMgr(MgrNum).LimitControl = ManagerLimitOff;
+
+				}
+				else if (SELECT_CASE_var == "FIXEDRATE") {
+					DemandMgr(MgrNum).LimitControl = ManagerLimitFixed;
+
+				}
+				else if (SELECT_CASE_var == "REDUCTIONRATIO") {
+					DemandMgr(MgrNum).LimitControl = ManagerLimitReductionRatio;
+
+				}
+				else {
+					ShowSevereError(CurrentModuleObject + "=\"" + cAlphaArgs(1) + "\" invalid value" + cAlphaFieldNames(3) + "=\"" + AlphArray(3) + "\".");
+					ShowContinueError("...value must be one of Off, Fixed, or Variable.");
+					ErrorsFound = true;
+				}}
+
+				DemandMgr(MgrNum).LimitDuration = NumArray(1);
+
+				DemandMgr(MgrNum).FixedRate = NumArray(2);
+				DemandMgr(MgrNum).ReductionRatio = NumArray(3);
+
+				DemandMgr(MgrNum).LowerLimit = NumArray(4);
+
+				// Validate Selection Control
+				{ auto const SELECT_CASE_var(AlphArray(4));
+				if (SELECT_CASE_var == "ALL") {
+					DemandMgr(MgrNum).SelectionControl = ManagerSelectionAll;
+
+				}
+				else if (SELECT_CASE_var == "ROTATEONE") {
+					DemandMgr(MgrNum).SelectionControl = ManagerSelectionOne;
+
+				}
+				else if (SELECT_CASE_var == "ROTATEMANY") {
+					DemandMgr(MgrNum).SelectionControl = ManagerSelectionMany;
+
+				}
+				else {
+					ShowSevereError(CurrentModuleObject + "=\"" + cAlphaArgs(1) + "\" invalid value" + cAlphaFieldNames(4) + "=\"" + AlphArray(4) + "\".");
+					ShowContinueError("...value must be one of All, RotateOne, or RotateMany.");
+					ErrorsFound = true;
+				}}
+
+				DemandMgr(MgrNum).RotationDuration = NumArray(5);
+
+				// Count number of string fileds for loading Controller:OutdoorAir names. This number must be increased in case if
+				// new string field is added or decreased if string fields are removed.
+				int AlphaShift = 4;
+
+				// Count actual pointers to air controllers
+				DemandMgr(MgrNum).NumOfLoads = 0;
+				for (LoadNum = 1; LoadNum <= NumAlphas - AlphaShift; ++LoadNum) {
+					LoadPtr = GetOAController(AlphArray(LoadNum + AlphaShift));
+					if (LoadPtr > 0) {
+						++DemandMgr(MgrNum).NumOfLoads;
+					}
+					else {
+						ShowSevereError(CurrentModuleObject + "=\"" + cAlphaArgs(1) + "\" invalid " + cAlphaFieldNames(LoadNum + AlphaShift) + "=\"" + AlphArray(LoadNum + AlphaShift) + "\" not found.");
+						ErrorsFound = true;
+					}
+				}
+
+				if (DemandMgr(MgrNum).NumOfLoads > 0) {
```

Comment:
Same as before. What about other demand managers? Should I add same error message there?